.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Can we remove this file from git? its super specific to your machine

Reply:
i had it in gitignore

Reply:
trying again

Reply:
but yes completely agree sorry :/

Reply:
@tommylees112  looks like this file is still here

Reply:
should be deleted now!